src/ethernetModule.cpp

```diff
@@ -56,12 +54,14 @@ void ethernetLoop() {
         Udp.write("Recieved UDP packet containing: ");
         Udp.write(recievedString);
         Udp.endPacket();
+        digitalWrite(LED_BUILTIN, 1);
```

Comment:
I don't like this tbh. For debugging I can understand that it is used here. But it should not be in the "production" code.